libs/npm/core/src/hooks/keycloak/useKeycloakEventHandler.ts

```diff
@@ -28,7 +28,7 @@ export const useKeycloakEventHandler = () => {
           state.setAuthReady(true);
         } else {
           //TODO: log error properly
-          console.debug(`keycloak event: ${eventType} error ${error}`);
+          console.debug(`keycloak event: ${eventType} error`, error);
```

Comment:
Hopefully it'll include the error information in the output.  This however shouldn't be a console.debug statement.